source/installation/install-docker.txt

```diff
@@ -0,0 +1,134 @@
+---------------
+
+- Relational Migrator is available on port 8080.
+
+- In the Docker container, Relational Migrator is installed to 
+  ``/opt/mongodb-relational-migrator/``.
+
+- When running in Docker, the ``user.properties`` file is ignored. Define any 
+  property overrides using environment variables. For example: 
+
+  .. code-block::
+
+     docker run -p 8080:8080 -e MIGRATOR_APP_TELEMETRY_ENABLE=false public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+- `Docker volumes <https://docs.docker.com/storage/volumes/>`__ can be used for 
+  persisting data and/or drivers if a container stops running or is restarted. 
+  You can create volumes to the paths:
+
+  - For data and logs: ``/root/Migrator``
+  - For drivers: ``/opt/mongodb-relational-migrator/lib/app/lib``
+
+  The following example runs Relational Migrator with persistent volumes for drivers 
+  and data:
+
+  .. code-block::
+
+     docker run -p 8080:8080 -v /migrator/drivers:/opt/mongodb-relational-migrator/lib/app/lib -v\
+     /migrator/data:/root/Migrator  public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+Before you Begin
+----------------
+
+You must have Docker installed on your computer. For more information, see  
+`Install Docker Engine <https://docs.docker.com/engine/install/>`__. 
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install a Kafka cluster
+
+      Use the following ``docker-compose`` configuration as an example to install a new 
+      Kafka cluster, or connect to an existing cluster.
+
+      .. code-block::
+
+         version: '3'
+         services:
+           kafka-1:
+           image: quay.io/debezium/kafka:2.2
+           ports:
+             - 19092:9092
+             - 19093:9093
+           environment:
+             CLUSTER_ID: oh-sxaDRTcyAr6pFRbXyzA
+             BROKER_ID: 1
+             KAFKA_CONTROLLER_QUORUM_VOTERS: 1@kafka-1:9093
+             NODE_ROLE: controller,broker
+           healthcheck:
+             test: ["CMD-SHELL", "./bin/kafka-topics.sh --version"]
+             interval: 5s
+             timeout: 10s
+             retries: 10
+
+   .. step:: Install a Kafka Connect cluster
+
+      Install a Kafka Connect cluster with the ``relational-migrator-connect`` image 
+      from the `public Amazon ECR <public.ecr.aws/v4d7k6c9/relational-migrator:latest>`__.
+
+   .. step:: Install Relational Migrator
+
+      You have two options for installing Relational Migrator:
+
+      1. From the `download center <https://www.mongodb.com/try/download/relational-migrator>`__, 
+         select the appropriate Relational Migrator installer.
+
+      #. Create a docker image using a Relational Migrator installer.
+
+         For a complete example, you can download the following ``docker-compose``: 
+         `Containerised kafka-connect cluster with Relational Migrator 
+         <https://migrator-installer-repository.s3-ap-southeast-2.amazonaws.com/1.4.3/docker-compose-migrator-single-kafka.yml>`__.
```

Comment:
This single Docker Compose file is probably the most important thing to include in the docs as it shows everything in one place. I'd suggest including this directly in the docs (we probably don't need the smaller compose fragments).
The download hyperlink is nice but it's tied to a specific (now out-of-date) version. @alwaysbemark can we publish a version of this with a stable URL that pulls the `:latest` tag?

Reply:
Updated to include the example. I left the current link at the moment but will update once a stable URL is available.

Reply:
We have a downloads MongoDB mirror now so we should use https://downloads.mongodb.org/migrator/1.5.0/docker-compose-migrator-single-kafka.yml here.
It probably makes sense to always have it versioned as things could change that could break the Docker compose file compatibility.

Reply:
@alwaysbemark It's fine to version the compose files, but we don't want to have to bake a version number into the docs as it will get out of date quickly; is there a stable URL for the latest version? If not, we should just link to the main download page (https://www.mongodb.com/try/download/relational-migrator) and make the compose files downloadable from the list.